docs/components/connectors/out-of-the-box-connectors/amazon-s3.md

```diff
@@ -92,6 +92,10 @@ The following JSON response is returned after a successful document upload opera
 - `key`: Echoes back the unique key of the uploaded document.
 - `link`: The document link.
 
+:::note
+Starting from version 8.7.0, the Amazon S3 Connector supports storing the response as a document. See additional details and limitations in [document handling](/components/concepts/document-handling.md).
+:::
+
```

Comment:
For S3, it's better to say something like "...the Amazon S3 Connector supports uploading documents from (or downloading documents to) the Camunda document store".